src/api/MainSolver.h

```diff
@@ -172,8 +172,14 @@ class MainSolver {
         [[nodiscard]] opensmt::span<const PTRef> getPreprocessedFormulas() const;
         [[nodiscard]] Logic::SubstMap getCurrentSubstitutions() const { return substitutions.current(); }
         void setSubstitutions(std::size_t level, Logic::SubstMap && subs) { substitutions.set(level, std::move(subs)); }
+        void prepareForProcessingFrame(std::size_t frameIndex);
 
     private:
+        std::size_t virtualFrameCount {1};
+        std::size_t actualFrameCount {1};
+        void pushInternal();
+        void popInternal();
```

Comment:
ordering within a class: 1. subclasses and types, 2. member functions, 3. member variables ?

Reply:
Fixed, if I understood it correctly.